### ISSUE TITLE
Add writable accessibility field to scenario form

### ISSUE THREAD

**Issue by lexvand** [nens/lizard-management-client#782]
Users should be able to change the accessibility of a scenario and its data. The backend developed a mechanism that changes the access_modifier of all related objects when it is PATCHed in API v4, so the user only has to manage the scenario (parent object) for this.

The field can be reused from for instance raster layers.

**Comment by hoanphungt** [comment]
@lexvand I deployed this one to staging. Tested it and now deployed it to production too. You can test it both on staging and production.
